Library build watch mode breaks if empty glob is passed to gulp.src

```
/home/deb/dev/osio/fabric8-planner/node_modules/gulp/node_modules/vinyl-fs/lib/src/index.js:20
    throw new Error('Invalid glob argument: ' + glob);
          ^
Error: Invalid glob argument: undefined
    at Gulp.src (/home/deb/dev/osio/fabric8-planner/node_modules/gulp/node_modules/vinyl-fs/lib/src/index.js:20:11)
    at Object.mach.transpileLESS (/home/deb/dev/osio/fabric8-planner/gulpfile.js:79:15)
    at FSWatcher.<anonymous> (/home/deb/dev/osio/fabric8-planner/gulpfile.js:130:12)
    at FSWatcher.emit (events.js:160:13)
    at FSWatcher.<anonymous> (/home/deb/dev/osio/fabric8-planner/node_modules/gulp/node_modules/chokidar/index.js:198:15)
    at FSWatcher._emit (/home/deb/dev/osio/fabric8-planner/node_modules/gulp/node_modules/chokidar/index.js:240:5)
    at FSWatcher.<anonymous> (/home/deb/dev/osio/fabric8-planner/node_modules/gulp/node_modules/chokidar/lib/nodefs-handler.js:263:16)
    at FSReqWrap.oncomplete (fs.js:167:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! fabric8-planner@0.0.0-development build: `gulp build "--watch"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fabric8-planner@0.0.0-development build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -122,16 +122,16 @@ gulp.task('build', function (done) {
   // watch
   if (argv.watch) {
     gulp.watch([appSrc + '/app/**/*.ts', '!' + appSrc + '/app/**/*.spec.ts']).on('change', function (e) {
-      util.log(util.colors.cyan(e.path) + ' has been changed. Compiling TypeScript.');
+      util.log(util.colors.cyan(e) + ' has been changed. Compiling TypeScript.');
       mach.transpileTS();
     });
-    gulp.watch([appSrc + '/app/**/*.less']).on('change', function (e) {
-      util.log(util.colors.cyan(e.path) + ' has been changed. Compiling LESS.');
-      mach.transpileLESS(e.path);
+    gulp.watch(appSrc + '/app/**/*.less').on('change', function (e) {
+      util.log(util.colors.cyan(e) + ' has been changed. Compiling LESS.');
+      mach.transpileLESS(e);
     });
-    gulp.watch([appSrc + '/app/**/*.html']).on('change', function (e) {
-      util.log(util.colors.cyan(e.path) + ' has been changed. Compiling HTML.');
-      mach.copyToDist(e.path);
+    gulp.watch(appSrc + '/app/**/*.html').on('change', function (e) {
+      util.log(util.colors.cyan(e) + ' has been changed. Compiling HTML.');
+      mach.copyToDist(e);
     });
     util.log('Now run');
     util.log('');

